tests/function-controller/main_test.go

```diff
@@ -16,12 +19,14 @@ type config struct {
 }
 
 func TestFunctionController(t *testing.T) {
+	rand.Seed(time.Now().UnixNano())
+
 	g := gomega.NewGomegaWithT(t)
 
 	cfg, err := loadConfig("APP")
 	failOnError(g, err)
 
-	restConfig, err := newRestClientConfig(cfg.KubeconfigPath)
+	restConfig := controllerruntime.GetConfigOrDie()
```

Comment:
then he is still able, just have this kubeconfig as your active one and it'll work

Reply:
in functionality nothing changed, user has even less things to do to have it work, but maybe I should document it

Reply:
but then again -> you can't run those tests on your own now, because there are internal calls to various services, so running it outside the cluster doesn't make sense

Reply:
Not outside - inside, but with another cluster :P